test/index.js

```diff
@@ -22,10 +22,10 @@ describe('babel-plugin-dynamic-import-node', () => {
       expect(result.trim()).to.equal(expected.trim());
     });
 
-    it(`works with ${folderName} and the ES2015 preset`, () => {
+    it(`works with ${folderName} and the env preset`, () => {
       const result = testPlugin(
         actual,
-        ['es2015'],
+        ['env'],
```

Comment:
hm, shouldn't instead `babel-preset-es2015` be a dev dep, and this file have two tests, one for env and one for es2015?